lib/models/Cat.js

```diff
@@ -0,0 +1,96 @@
+const pool = require('../utils/pool');
+
+module.exports = class Cat {
+  id;
+  name;
+  age;
+  color;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.age = row.age;
+    this.color = row.color;
+  }
+
+  static async insert({ name, age, color }) {
+    const { rows } = await pool.query(
+      `
+      INSERT INTO 
+      cats (name, age, color)
+      VALUES 
+      ($1, $2, $3)
+      RETURNING
+      *
+      `,
+      [name, age, color]
+    );
```

Comment:
We might add an early return for all of these static methods to prevent some errors in the case of the query not returning anything from the database. Something like `if!(rows[0] return null`